src/services/programService/httpClient.js

```diff
@@ -27,14 +27,292 @@ import fetch from 'node-fetch';
+			}
+		});
+};
+
+export const listCancers = async (jwt = null) => {
+	const url = `${PROGRAM_SERVICE_HTTP_ROOT}/programs/cancers`;
+	return await fetch(url, {
 		method: 'get',
+		headers: {
+			Authorization: `Bearer ${jwt}`,
+		},
 	})
 		.then(restErrorResponseHandler)
+		.then((response) => response.json())
+		.then((data) => {
+			if (data && Array.isArray(data)) {
+				return formatAndSortMultipleLists(data);
+			} else {
+				console.log('Error: no data is returned from /programs/cancers');
+				throw new Error('Unable to retrieve cancers data.');
+			}
+		});
+};
+
+export const listInstitutions = async (jwt = null) => {
+	const url = `${PROGRAM_SERVICE_HTTP_ROOT}/programs/institutions`;
+	return await fetch(url, {
+		method: 'get',
+		headers: {
+			Authorization: `Bearer ${jwt}`,
+		},
+	})
+		.then(restErrorResponseHandler)
+		.then((response) => response.json())
+		.then((data) => {
+			if (data && Array.isArray(data)) {
+				return formatAndSortMultipleLists(data);
+			} else {
+				console.log('Error: no data is returned from /programs/institutions');
+				throw new Error('Unable to retrieve institutions data.');
+			}
+		});
+};
+
+export const listPrimarySites = async (jwt = null) => {
+	const url = `${PROGRAM_SERVICE_HTTP_ROOT}/programs/primarySites`;
+	return await fetch(url, {
+		method: 'get',
+		headers: {
+			Authorization: `Bearer ${jwt}`,
+		},
+	})
+		.then(restErrorResponseHandler)
+		.then((response) => response.json())
+		.then((data) => {
+			if (data && Array.isArray(data)) {
+				return formatAndSortMultipleLists(data);
+			} else {
+				console.log('Error: no data is returned from /programs/primarySites');
+				throw new Error('Unable to retrieve primarySites data.');
+			}
+		});
+};
+
+export const listRegions = async (jwt = null) => {
+	const url = `${PROGRAM_SERVICE_HTTP_ROOT}/programs/regions`;
+	return await fetch(url, {
+		method: 'get',
+		headers: {
+			Authorization: `Bearer ${jwt}`,
+		},
+	})
+		.then(restErrorResponseHandler)
+		.then((response) => response.json())
+		.then((data) => {
+			if (data && Array.isArray(data)) {
+				return formatAndSortMultipleLists(data);
+			} else {
+				console.log('Error: no data is returned from /programs/regions');
+				throw new Error('Unable to retrieve regions data.');
+			}
+		});
+};
+
+export const listCountries = async (jwt = null) => {
+	const url = `${PROGRAM_SERVICE_HTTP_ROOT}/programs/countries`;
+	return await fetch(url, {
+		method: 'get',
+		headers: {
+			Authorization: `Bearer ${jwt}`,
+		},
+	})
+		.then(restErrorResponseHandler)
+		.then((response) => response.json())
+		.then((data) => {
+			if (data && Array.isArray(data)) {
+				return formatAndSortMultipleLists(data);
+			} else {
+				console.log('Error: no data is returned from /programs/countries');
+				throw new Error('Unable to retrieve countries data.');
+			}
+		});
+};
+
+export const createProgram = async (program, jwt = null) => {
```

Comment:
@joneubank This is the createProgram function. The formattedProgram is the format in swagger.